syntax = "proto3";

package service;

service Service {
  rpc GetServerInfo(ServerRequest) returns (ServerResponse);
}

message ServerRequest {
  string query = 1;
}

message ServerResponse {
  string info = 1;
}
